avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -207,14 +213,15 @@ public Object convert(Schema schema, Object value) {
     });
   }
 
+  static final String AVRO_PROP = "avro";
   static final String AVRO_LOGICAL_TYPE_PROP = "logicalType";
   static final String AVRO_LOGICAL_TIMESTAMP_MILLIS = "timestamp-millis";
   static final String AVRO_LOGICAL_TIME_MILLIS = "time-millis";
   static final String AVRO_LOGICAL_DATE = "date";
   static final String AVRO_LOGICAL_DECIMAL = "decimal";
   static final String AVRO_LOGICAL_DECIMAL_SCALE_PROP = "scale";
   static final String AVRO_LOGICAL_DECIMAL_PRECISION_PROP = "precision";
-  static final String CONNECT_AVRO_DECIMAL_PRECISION_PROP = "avro.precision";
+  static final String CONNECT_AVRO_DECIMAL_PRECISION_PROP = "connect.decimal.precision";
```

Comment:
This i think links with another query you raised. This is a connect specific prop, as such should avoid using the avro. namespace and use connect. namespace as used else where. On this note though we even questioned its usefulness/thought of removing entirely as actually this information is held in "precision" prop in Avro spec anyhow.